### PR TITLE
Update index.mdx

### PR DESCRIPTION
**Pull request by ljcarot**
### Related Ticket(s)

{{Provide url(s) to the related ticket(s) that this pull request addresses}}

### Description

{{Add a human-readable description / detail summary of what the PR is changing and any details around how and why}}

{{If applicable, include a screenshot indicating an example or examples of what the PR is changing in the application}}

### Changelog

**New**

- {{new thing}}

**Changed**

- {{changed thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

**Comment by JennySanchez** [comment]
@ljcarot I noticed there are two "older" in the notification message.
Screen Shot 2021-03-04 at 8 23 38 AM

File: src/pages/about/index.mdx [hunk 1]
```diff
@@ -10,13 +10,15 @@ IBM.com is a place for people to learn about, try, and buy IBM products and serv
...
+Towards that end, Carbon for IBM.com has been built with the goal of providing efficiency and consistency across IBM.com using IBM Design Language (IDL) principles and based on the Carbon Design System (Carbon).  
+
+This site is intended to provide guidance and resources for designers and developers who create sites and/or applications on IBM.com and other IBM domains.
 
 </PageDescription>
 
 <InlineNotification>
 
-**Note:** This site is intended to provide guidance and resources for designers and developers who create sites and/or applications on IBM.com and other IBM domains.
+**Note:** Carbon for IBM.com is the strategic design system for IBM web experiences.  Any application using older older systems like Northstar (v18), v19, or even earlier versions should make plans to adopt Carbon for ibm.com by the end of 2021.  After all applications have migrated, plans and dates will be announced for sunsetting the older systems.  Reach out to us in our IBM slack channel at [#carbon-for-ibm-dotcom](https://ibm-studios.slack.com/archives/C2PLX8GQ6) with any questions.
```
Comment (JennySanchez) [inline src/pages/about/index.mdx:22]: **Note:** Carbon for IBM.com is the strategic design system for IBM web experiences.  Any application using older systems like Northstar (v18), v19, or even earlier versions should make plans to adopt Carbon for ibm.com by the end of 2021.  After all applications have migrated, plans and dates will be announced for sunsetting the older systems.  Reach out to us in our IBM slack channel at [#carbon-for-ibm-dotcom](https://ibm-studios.slack.com/archives/C2PLX8GQ6) with any questions.

**Comment by ljcarot** [comment]
@JennySanchez Can we just merge the latest version?  I realized after someone was looking for message about Sunsetting v18 that we didn't have it clearly highlighted.

**Review by JennySanchez — APPROVED** [review]
Thank you, Linda.

**Pull request merged**